mscore/exportxml.cpp

```diff
@@ -3762,7 +3789,7 @@ static int findTrackForAnnotations(int track, Segment* seg)
       }
 
 //---------------------------------------------------------
-//  repeatAtMeasureStart -- write repeats at begin of measure
+//  repeatAtMeasureStart -ite repeats at begin of measure
```

Comment:
https://musescore.org/en/developers-handbook/musescore-coding-rules#Coding-Style can help, and the qt2-musescore.xml from there can even automate this in QtCreator